requirements.txt

```diff
@@ -1,7 +1,7 @@
 torch>=1.7  # TODO: regenerate weights with lower PT version
 torchmetrics
 torchvision>=0.8  # TODO: lower to 0.7 after PT 1.6
-pytorch_lightning @ git+https://github.com/PyTorchLightning/pytorch-lightning.git
+pytorch_lightning==1.3.0rc1
```

Comment:
careful, when we install both lightning and flash locally for development, this could be a problem?

Reply:
We need this release for Flash actually :)